docs/source/guides/coding_style.rst

```diff
@@ -0,0 +1,96 @@
+Coding style and rules
+============================
+
+The following rules and notation come from examples in PR-revisions that we think can become common rules. Each rule will have an example and a description.
+
+
+Commenting out code
+-------------------------
+
+Having commented code makes no sense. It should be eliminated from the commit altogether. If for some reason you fear losing the piece of code in the future, remember that it will be present in the git history (if it has taken part of the master branch at some point):
```

Comment:
I would make a little more emphasis on the if it has taken part of the master branch at some point, maybe having instead of having it between parenthesis, something like this:

"[...] git history, if it has taken part of the **master branch** at some point:"